sdk/communication/azure-communication-administration/README.md

```diff
@@ -28,15 +28,39 @@ Acquired phone numbers can come with many capabilities, depending on the country
 
 ## Key concepts
 
-To use the Admnistration SDK, a resource access key is required for authentication. 
+There are two forms of authentication to use the Admnistration SDK.
 
+### Using your AAD Credentials
```

Comment:
To make the page more comprehensive, this header should read similar to the one below, "Access Key Authentication", e.g. "Active Directory Token Authentication"